content/account/app/settings.textile

```diff
@@ -0,0 +1,49 @@
+| *Enabled* | When an application is *disabled*, it no longer accepts new connections and deactivates all associated services. Clients cannot connect, send or receive messages, or use other Ably services. When *enabled*, the application allows new connections and activates all related services. |
+
+h3. Channel rule configuration
+
+The following explains the configuration rules for specific "namespaces":/channels#namespaces or "channels:":/channels
+
+|_. Section |_. Description |
+| *Namespace or channel ID* | Identify the specific namespace or channel to which this rule will apply. |
+| *Persist last message* | Stores the last message published on a channel for 365 days, accessible via the rewind mechanism. |
+| *Persist all messages* | Ably stores all messages for two minutes by default. Depending on your account package, this can be increased to 24 or 72 hours. It is also possible to persist the last message sent to a channel for a year. |
+| *Identified* | Requires clients to authenticate with a client ID to interact with channels in this namespace. |
+| *TLS only* | Restricts access to channels within this namespace to clients connected using TLS. |
+| *Push notifications enabled* | Enables publishing messages with a push payload, triggering native push notifications to registered devices. |
+| *Message interactions enabled* | Enables unique time serial fields in messages, enabling typed references between messages (e.g., implementing 'Likes' in a chat). |
+| *Server-side batching enabled* | Batches inbound messages on the server side before sending them to subscribers based on the configured policy. |
+| *Cancel* | Discards changes and closes the dialog without saving the new rule.                                                                                    |
+
+h3. Protocol adapter settings
+
+The following explains the configuration support for various communication protocols ("Pusher":/protocols/pusher, "PubNub":/protocols/pubnub, "MQTT":/protocols/mqtt), enabling different client libraries to interact with Ably.
+
+|_. Protocol Adapter Settings  |_. Description |
+| *Pusher protocol support* | Provides compatibility with the Pusher protocol. |
+| *PubNub protocol support* | Provides compatibility with the PubNub protocol. |
+| *MQTT protocol support* | Provides compatibility with the MQTT protocol. |
+
+h3. Channel lifecycle and occupancy events
+
+The following explains managing notifications for channel "lifecycle":/metadata-stats/metadata/subscribe?lang=javascript#channel-lifecycle events and occupancy updates, enabling realtime insights into channel activity.
+
+|_. Channel Lifecycle and Occupancy Events |_. Description |
+| *Enable occupancy* | The toggle will enable notifications for channel lifecycle events and occupancy updates. |
```

Comment:
See comment on the overview about this. I think this is to enable https://ably.com/docs/channels/options#subscribe-occupancy